heron/common/src/java/com/twitter/heron/common/config/SystemConfig.java

```diff
@@ -540,6 +546,12 @@ public int getMetricsMgrNetworkOptionsSocketSendBufferSizeBytes() {
             SystemConfig.METRICSMGR_NETWORK_OPTIONS_SOCKET_SEND_BUFFER_SIZE_BYTES));
   }
 
+  public int getHeronMetricsMaxExceptionsCount() {
+    Object value = this.config.get(
+        SystemConfig.HERON_METRICS_MAX_EXCEPTIONS_COUNT);
+    return value == null ? Integer.MAX_VALUE : TypeUtils.getInteger(value);
```

Comment:
This pattern is repeated in this class. Could we create a methods like this in this class to simplify this:
```
public static int getValue(Map<String, Object> config, String key, Integer default);
```
might be better to do in another pr.